unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldBridge/BuilderInWorldBridge.cs

```diff
@@ -16,6 +16,9 @@
 /// </summary>
 public class BuilderInWorldBridge : MonoBehaviour
 {
+    public event Action OnPublishOK;
+    public event Action<string> OnPublishKO;
```

Comment:
```suggestion
    public event Action<string> OnPublishOK;
```

Reply:
This suggestion is to also send a string with the result "OK"?

Reply:
I think Brian thought `OnPublishKO` was a typo. I agree the naming is confusing, but renaming to `OnPublishOK` wouldn't work because we already have that declared right above.